test/parallel/test-accessor-properties.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
-require('../common');
+const common = require('../common');
 
+if (!common.hasCrypto)
+  common.skip('missing crypto');
```

Comment:
Rather than skipping this entire test, can we move the `crypto` stuff to its `if` block and only run those tests if `hasCrypto` is true? This way, we aren't skipping the non-crypto tests in the file.

Reply:
Sounds good, I'll do that.